ee/control/control.go

```diff
@@ -147,15 +147,6 @@ func (cs *ControlService) requestIntervalChanged(interval time.Duration) {
 		return
 	}
 
-	// perform a fetch now
-	if err := cs.Fetch(); err != nil {
```

Comment:
I think we might still want the immediate fetch if it doesn't come from control server (comes from menubar / local server). What using a trylock inside the fetch function and exiting if we can't get lock?

Reply:
Oh, thank you, that's a really good point. It looks like that will work -- I'll document and push up that change.